src/fakeelliptical.cpp

```diff
@@ -11,7 +10,6 @@
 #ifdef Q_OS_ANDROID
 #include <QLowEnergyConnectionParameters>
 #endif
-#include "keepawakehelper.h"
```

Comment:
Required for Android build.

Reply:
I've restored these but put them inside the #ifdef Q_OS_ANDROID  to avoid the QtCreator warning about unused headers.
Have left the ones I didn't remove.